templates/dotnet/src/Appwrite/Client.cs.twig

```diff
@@ -172,10 +158,24 @@ namespace {{ spec.title | caseUcfirst }}
                     request.Headers.Add(header.Key, header.Value);
                 }
             }
-            HttpResponseMessage httpResponseMessage = await http.SendAsync(request);
+            try
+            {
+                var httpResponseMessage = await http.SendAsync(request);
+                var code = (int) httpResponseMessage.StatusCode;
+                var response = JObject.Parse(await httpResponseMessage.Content.ReadAsStringAsync());
+                
+                if (code >= 400) {
+                    var message = response["message"].ToString();
+                    throw new AppwriteException(message, code, response.ToString());
```

Comment:
Avoid mentioning the name `Appwrite` use the twig param, you can use references from other @lohanidamodar PRs.

Reply:
hehe, missed these one 😄